ui/src/App.tsx

```diff
@@ -11,7 +11,7 @@ function App() {
   const { themeClass } = useTheme();
 
   return (
-    <ThemeProvider>
+    <ThemeProvider forceColorMode="dark">
```

Comment:
@amessbee observed an issue with some UI elements showing up on light mode, wondering how this renders. I can try it out real quick

Reply:
It's kind of tricky because we're using daisyui as well as interchain-ui. This makes interchain-ui dark (including the network dropdown), and the `ui/tailwind.config.js` change makes daisyui dark (majority of the ui elements).

Reply:
Hmm it doesn't seem to be working for me though, the network dropdown still looks light, looking into it